.claude/skills/document-skills/docx/ooxml/scripts/validation/docx.py

```
@@ -0,0 +1,274 @@
+from .base import BaseSchemaValidator
+
+
+class DOCXSchemaValidator(BaseSchemaValidator):
+    """Validator for Word document XML files against XSD schemas."""
+
+    # Word-specific namespace
+    WORD_2006_NAMESPACE = "http://schemas.openxmlformats.org/wordprocessingml/2006/main"
+
+    # Word-specific element to relationship type mappings
+    # Start with empty mapping - add specific cases as we discover them
+    ELEMENT_RELATIONSHIP_TYPES = {}
```

Ruff (0.14.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)